src/main/java/seedu/lifeasier/ui/Ui.java

```diff
@@ -92,10 +105,98 @@ public String readCommand() {
         return conversation.nextLine();
     }
 
+    public int readSingleIntInput() {
+        return Integer.parseInt(readCommand());
+    }
+
+    public void showSelectTaskToEdit(String type) {
+        System.out.println("Please select the " + type + " you want to edit.");
+    }
+
+    public void showSelectParameterToEdit() {
+        System.out.println("Please select the parameter you want to edit.");
+    }
+
+    public void showSelectTaskToDelete(String type) {
+        System.out.println("Please select the " + type + " you want to delete.");
+    }
+
+    public void showInputFormat(String type) {
+        switch (type) {
+
+        case (PARAM_EVENT):
+            System.out.println("Please input your new time in this format: " + NEW_EVENT_TIME_INPUT_FORMAT);
+            break;
+
+        case (PARAM_DEADLINE):
+            System.out.println("Please input your new time in this format: " + NEW_DEADLINE_TIME_INPUT_FORMAT);
+            break;
+
+        case (PARAM_LESSON):
+            System.out.println("Please input your new time in this format: " + NEW_LESSON_TIME_INPUT_FORMAT);
+            break;
+
+        default:
+            break;
+        }
+    }
+
+    public void showInputMessage(String type) {
+        switch (type) {
+
+        case (PARAM_LESSON):
+            System.out.println("Please input your new Module Code");
+            break;
+
+        case (PARAM_DEADLINE):
+            System.out.println("Please input your new Deadline name");
+            break;
+
+        case (PARAM_EVENT):
+            System.out.println("Please input your new Event name");
+            break;
+
+        default:
+            break;
+        }
+    }
+
+    public void showEditableParameters(String type) {
```

Comment:
Could you consider renaming this to include "Message" for consistency?
```suggestion
    public void showEditableParametersMessage(String type) {
```